src/ntpd.c

```diff
@@ -936,6 +942,11 @@ static int ntpd_read (void)
 		return (-1);
 	}
 
+	if (ntohs(ik->status) & STA_NANO) {
```

Comment:
No, it isn't. ntpd just directly sends the struct obtained from ntp_adjtime syscall.